doc/README.gpu_support

```diff
@@ -0,0 +1,128 @@
+# Open-CE GPU Support
+
+The Open-CE build scripts and feedstocks support CUDA from NVIDIA. In order
+for the build scripts to work, CUDA must be installed on the system on which
+the builds are taking place. This can be accomplished in two ways (see below).
+
+**In all cases, the `CUDA_HOME` environment variable must be set to the base
+directory where CUDA has been installed.**
+
+
+## Bare metal builds
+
+If you are using Open-CE on a bare metal system. Install the appropriate version
+of CUDA on the server that matches the version of Open-CE you are using. There
+is a reference table below.
+
+| Open-CE version | CUDA version |
+|-----------------|--------------|
+| 1.0 (marmotini) | 10.2         |
+|                 |              |
+
+It's important to install CUDA correctly. This procedure is dependent on the system
+type and CUDA version. It's advised to follow NVIDIA's [official installation documentation](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html)
+Note that NVLINK2-enabled POWER9 systems require a few [extra steps](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#power9-setup)
+
+
+## Container builds
+
+It is recommended to run the Open-CE build scripts within a container. NVIDIA provides
+official CUDA base containers [on](https://hub.docker.com/r/nvidia/cuda) [dockerhub](https://hub.docker.com/r/nvidia/cuda-ppc64le),
+which are ideal for ensuring a properly installed CUDA environment. 
+
+Open-CE provides a working set of images that will work for running CUDA enabled builds.
```

Comment:
We should mention that these are based on the Nvida images mentioned above, otherwise people might think they should use the Nvidia ones as opposed to the ones that we have created.